rhwp/rhwp-agent/agents/claude.mjs

```
@@ -680,15 +924,19 @@ export function createClaudeSession(opts, {
-          });
+          if (nativeUserInput) dispatchNative(text);
+          else dispatchLegacy(text);
         } catch (e) {
+          if (nativeUserInput) {
+            nativeUserInput = false;
+            process.stderr.write(`[claude] native user-input transport unavailable; using MCP fallback: ${e?.message ?? e}\n`);
+            try {
+              dispatchLegacy(text);
+              return;
+            } catch (fallbackError) {
+              e = fallbackError;
```

Biome (2.5.7)

[error] Reassigning a catch parameter is confusing.

(lint/suspicious/noCatchAssign)